cmd/serve.go

```diff
@@ -2,44 +2,42 @@ package cmd
 
 import (
 	"embed"
-	"fmt"
 	"log"
-	"net/http"
 	"os"
 	"path/filepath"
 
-	"github.com/labstack/echo/v4"
-	"github.com/labstack/echo/v4/middleware"
+	"github.com/nathanjisaac/actual-server-go/internal"
 	"github.com/nathanjisaac/actual-server-go/internal/core"
-	"github.com/nathanjisaac/actual-server-go/internal/routes"
-	"github.com/nathanjisaac/actual-server-go/internal/storage/sqlite"
 	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
 )
 
 var BuildDirectory embed.FS
 
 // serveCmd represents the serve command
 var serveCmd = &cobra.Command{
 	Use:   "serve",
-	Short: "A brief description of your command",
-	Long: `A longer description that spans multiple lines and likely contains examples
-and usage of using your command. For example:
-
-Cobra is a CLI library for Go that empowers applications.
-This application is a tool to generate the needed files
-to quickly create a Cobra application.`,
+	Short: "This command will start the actual-sync server",
+	Long: `This command will start the actual-sync server with the 
+specified configurations along with this command.`,
 	Run: func(cmd *cobra.Command, args []string) {
-		serverFiles, err := filepath.Abs("data/server-files")
-		if err != nil {
-			log.Fatal(err)
-		}
-
-		userFiles, err := filepath.Abs("data/user-files")
```

Comment:
Did we still want to create the user files folder?

Reply:
Yes. The message databases is supposed to be saved inside this folder according to the original code. This db is created and accessed under the `/sync` endpoint.